tstest/integration/vms/gen/test_codegen.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// build ignore
+
+package main
+
+import (
+	_ "embed"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	. "github.com/dave/jennifer/jen"
```

Comment:
The [upstream examples](https://github.com/dave/jennifer#jennifer) suggest the use of a dot import, so I use one here.

Reply:
I think it's preferable not to do that anyway; it doesn't look like it'll make the code much more annoying to write.